currencyii/src/main/java/nl/tudelft/trustchain/currencyii/ui/bitcoin/DAOCreateFragment.kt

```diff
@@ -55,9 +56,14 @@ class DAOCreateFragment : BaseFragment() {
         }
 
         generate_new_seed.setOnClickListener {
-            val params = when (production_testnet_input.isChecked) {
-                true -> BitcoinNetworkOptions.TEST_NET
-                false -> BitcoinNetworkOptions.PRODUCTION
+            val params = when (bitcoin_network_radio_group.checkedRadioButtonId) {
+                1 -> BitcoinNetworkOptions.PRODUCTION
+                2 -> BitcoinNetworkOptions.TEST_NET
+                3 -> BitcoinNetworkOptions.REG_TEST
+                else -> {
```

Comment:
same here radio buttons are designed to have one selected just make one default and drop the else